backend/protos/mgmt/v1alpha1/transformer.proto

```diff
@@ -501,4 +519,9 @@ service TransformersService {
   rpc IsTransformerNameAvailable(IsTransformerNameAvailableRequest) returns (IsTransformerNameAvailableResponse) {}
   rpc ValidateUserJavascriptCode(ValidateUserJavascriptCodeRequest) returns (ValidateUserJavascriptCodeResponse) {}
   rpc ValidateUserRegexCode(ValidateUserRegexCodeRequest) returns (ValidateUserRegexCodeResponse) {}
+
+  // Retrieve a list of available Pii recognizers for use with the TransformPiiText transformer
+  rpc GetTransformPiiRecognizers(GetTransformPiiRecognizersRequest) returns (GetTransformPiiRecognizersResponse) {
```

Comment:
I put this here because it seemed more generic than just shoving it in the anonymization service since it is specific to the transformPiitext transformer.
wdyt @alishakawaguchi, @evisdrenova  ?

Reply:
Is this referring to the getEntities endpoint that we talked about or something else? If so, are we just using recognizers instead of entities everywhere? I like entities because that's industry standard i.e. Named Entity Recognition model.

Reply:
oh yeah actually good call, I'm surfacing the wrong function. I should be surfacing the entities, not the entity recognziers.

Reply:
That is kind of beside my original question though.

Reply:
yeah back to your original point - IMO i'd put it in the anonymizer service since it retrieves the entities from the presidio api and functionally is currently only used for anonymization even if it is consumed by a transformer

Reply:
only problem is that ceases to be true once we wire it up to the async job flow. it's a supporting value for a transformer

Reply:
which i think is okay imo - you're still getting those values from presidio it's just being used by the transformer, for ex. we have the connection rpcs in the connection_data service even though it's an input into a job. but i think you can go either way with it

Reply:
I think the transformer service makes the most sense